numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -3744,7 +3744,7 @@ def add_newdoc(place, name, doc):
 
 add_newdoc('numpy.core.umath', 'gcd',
     """
-    Returns the greatest common divisor of |x1| and |x2|
+    Returns the greatest common divisor of x1 and x2
```

Comment:
This is a change in meaning, but I think it's still correct.

Reply:
whoops, that should still be abs, sphinx (and me both) took it to mean a substitution. Fixing